### ISSUE TITLE
Learning rate set when using 2 GPUs to train

### ISSUE THREAD

**Issue by skyeyeCV** [AlexeyAB/darknet#7274]
@AlexeyAB  Hello, when I set the learning_rate in yolov3.cfg file to be 0.001 and use 2 GPUs to train yolov3 model，the learning rate which the training log displays is 0.002000 rate but not 0.001000 rate. If I want to set the actual learning rate to be 0.001 when using 2 GPUs to train yolov3 model, should I set the parameter learning_rate in yolov3.cfg file to be 0.001 or 0.0005? Thanks!
